apps/journeys-admin/src/components/JourneyList/JourneyCard/JourneyCardMenu/JourneyCardMenu.tsx

```diff
@@ -130,6 +134,7 @@ export function JourneyCardMenu({
   }
   const handleCloseMenu = (): void => {
     setAnchorEl(null)
+    if (onMenuClose) onMenuClose()
```

Comment:
```suggestion
 onMenuClose?.()
```

could do something like this

Reply:
I never knew that shorthand existed! Very cool. Added.